src/app/pages/apptrust/apptrust.page.ts

```diff
@@ -15,18 +16,27 @@ declare var navigator: any;
 
 export class AppTrustPage implements OnInit {
   private static readonly METRICS_KEY = 'security';
+  private platform: Platform;
+  public serverUrl;
+  public app;
+
+  constructor(private securityService: SecurityService) {
+
+  }
+
 
   ngOnInit() {
   }
 
   public isAvailable() {
-      // return this.platform.is('cordova');
-      return true;
+    //return this.platform.is('cordova');
+    return true;
```

Comment:
Kinda pointless (this is used in UI as far I remember)

Reply:
disable while debugging , will turn back on. :)